src/03-arrays.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * The $input variable contains an array of digits
+ * Return an array which will contain the same digits but repetitive by its value
+ * without changing the order.
+ * Example: [1,3,2] => [1,3,3,3,2,2]
+ *
+ * @param  array  $input
+ * @return array
+ */
+function repeatArrayValues(array $input)
+{
+    $newArr = array();
+
+    foreach ($input as $value):
```

Comment:
The alternative foreach syntax is usually used for templates, where PHP and HTML is mixed.